packages/gcp-logging/src/index.ts

```diff
@@ -80,13 +80,30 @@ export const customLogger = (
         [k]: v,
       }))
 
+      let messageOrCause = message
+      let exception: any = undefined
+
+      if (cause) {
+        messageOrCause = message ?? Cause.pretty(cause)
+        switch(cause._tag){
+          case 'Fail':
+            exception = Cause.pretty(cause)
+            break
+          case 'Die':
+            exception = Option.getOrElse(Cause.dieOption(cause), () => {stack: 'Exception came with no defect!'})
+            exception = exception.stack
+            break
+        }
+      }
+
+      // TODO: should we use logAnnotations for span & trace?
```

Comment:
It seems that we can retrieve annotations via `Effect.logAnnotations`. Shall we adjust? was a particular reason not to? (I have not tried)

Reply:
I'm still not clear, how would that be used. Can you reference some docs?